src/main/java/com/rultor/agents/ecs/StartsECS.java

```diff
@@ -79,14 +82,14 @@ public StartsEC2(final Amazon amaz) {
     @Override
     //@todo #629 Add Instance params to Directive, for example publicIpAddress
     public Iterable<Directive> process(final XML xml) throws IOException {
-        final Instance instance = this.amazon.runOnDemand();
+        final ContainerInstance instance = this.amazon.runOnDemand();
         Logger.info(
             this,
-            "EC2 instance %s created",
+            "ECS instance %s created",
             instance
         );
         return new Directives().xpath("/talk")
-            .add("ec2")
-            .attr("id", instance.getInstanceId());
+            .add("ecs")
+            .attr("id", instance.getEc2InstanceId());
```

Comment:
@xupyprmv @mkordas in fact please **do not** change `ec2` to `ecs` as the directive index. I know its a little less visually pleasing , but the table definitions are what they are in Dynamo and I'd rather not change those here.
No value in doing so right now imo :)